deployment/conf/.templates/condor_config.templ

```diff
@@ -1,29 +1,75 @@
-##  What machine is your central manager?
+######################################################################
```

Comment:
I'm guessing this condor_config comes from the rpm instead of from andypohl?  Are there major differences?  For example I don't see UID_DOMAIN or DAEMON_LIST in here, but maybe they have a reasonable default.

Reply:
Deleted unused files

Reply:
What about condor_config.templ and the different variables in the two versions?